Makefile

```diff
@@ -55,11 +55,11 @@ install-travis:
 # Example:
 #   make check
 #   make test
-#   make check WHAT=pkg/build TESTFLAGS=-v
-check:
+#   make check WHAT=pkg/docker TESTFLAGS=-v
+test check:
```

Comment:
the "check" target should run the build, verify, and test targets.
the "test" target should run the hack/test-go logic.